src/internal/utils.ts

```diff
@@ -3,20 +3,22 @@
  * @param value
  * @returns
  */
-export function isArrayBuffer(value: any): value is ArrayBuffer {
+export function isArrayBuffer(value: unknown): value is ArrayBuffer {
```

Comment:
wow :face_with_head_bandage:

Reply:
Yup, I'm sure there's something more clever to do there, but in the meantime, it will do. 

What I learned in the process is that the `unknown` type is less permissive than `any` and will force you to do type assertions whenever you're using that value. See [this article](https://mariusschulz.com/blog/the-unknown-type-in-typescript) that I found useful.

Reply:
I'm much more concerned about the current inability to make type checking on the `crypto.hmac` calls though to be fair 👍🏻 

(looking into switching to webcrypto there down the road)